lte/gateway/deploy/roles/magma/files/magma-bridge-reset.sh

```diff
@@ -109,10 +113,13 @@ then
   ip link set dev "$SGI_PORT"  up
 fi
 
-# restore OVS flows
-ovs-ofctl del-flows "$SGI_BR"
-ovs-ofctl add-flows "$SGI_BR" "$FLOW_DUMP"
-rm "$FLOW_DUMP"
+if [ "$KMOD_RELOAD" != '-n' ];
+then
+  # restore OVS flows
+  ovs-ofctl del-flows "$SGI_BR"
+  ovs-ofctl add-flows "$SGI_BR" "$FLOW_DUMP"
+  rm "$FLOW_DUMP"
+fi
```

Comment:
the flow table should be restored in all cases, since we have lost flow table `ifdown $SGI_BR0` was done.

Reply:
ah okay I miss-understood you then , updated